lib/current_order.rb

```diff
@@ -0,0 +1,32 @@
+class CurrentOrder
+
+  attr_reader :order
+
+  def initialize
+    @order = []
+  end
+
+  def add_item(item)
+    @order << item
+  end
+
+  def total
```

Comment:
Very minor thing: `#sum` can be used here to get the total price in one line...

`@order.sum { |item| item[:price] }`

What do you think?  Concision does not always equal quality...